easybuild/easyconfigs/l/libreadline/libreadline-6.3-GCC-4.8.2.eb

```diff
@@ -10,6 +10,7 @@ description = """The GNU Readline library provides a set of functions for use by
  to recall and perhaps reedit those lines, and perform csh-like history expansion on previous commands."""
 
 toolchain = {'name': 'GCC', 'version': '4.8.2'}
+toolchainopts = {'pic': True}
```

Comment:
all `libreadline` and `zlib` easyconfigs should have `pic` enabled, I found a few that don't...